src/parser/parser.py

```diff
@@ -0,0 +1,64 @@
+"""Parses a PDF document.
```

Comment:
What is the best convention in file naming? For classes should you capitalize the file, as in `Parser.py`, or is it different than other object-oriented programming languages?

I am using C++ as a benchmark, so it's very likely that it applies different in Python. But Idk.

Reply:
module_name, package_name, ClassName, method_name, ExceptionName, function_name, GLOBAL_CONSTANT_NAME, global_var_name, instance_var_name, function_parameter_name, local_var_name.